modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/site_settings/index.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import ClayPanel from '@clayui/panel';
+import React, {useMemo, useState} from 'react';
+
+import AddIconPackModal from './AddIconPackModal';
+
+export default function SiteIconPackConfiguration({
+	allIconResourcePacks: icons,
+	portletNamespace,
+	saveSiteIconPacksURL,
+	siteIconResourcePacks,
+}) {
+	const [selectedPacks, setSelectedPacks] = useState(siteIconResourcePacks);
+	const [searchQuery, setSearchQuery] = useState('');
+	const [showAddModal, setShowAddModal] = useState(false);
+
+	const iconPackNames = Object.keys(icons);
+
+	const filteredIcons = useMemo(() => {
+		return iconPackNames.reduce((acc, packName) => {
+			return {
+				...acc,
+				[packName]: {
+					...icons[packName],
+					icons: icons[packName].icons
+						.filter((icon) => Object.keys(icon).length)
+						.filter((icon) =>
+							icon.name
+								.toLowerCase()
+								.includes(searchQuery.toLocaleLowerCase())
+						),
```

Comment:
Hey @bryceosterhaus, why not:

```suggestion
						
						.filter((icon) =>
						        Object.keys(icon).length &&
							icon.name
								.toLowerCase()
								.includes(searchQuery.toLocaleLowerCase())
						),
```

Reply:
hmm, I actually don't really remember why I added it this way. I will take another look